### PR TITLE
Grand rick001 patch 5

### PR DESCRIPTION
**Pull request by grand-rick001**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5180"></a>

### CODE REVIEW

File: data/grand-rick001.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "Patrick Murimi",
   "type": "personal",
   "bio": "I am a Junior Full Stack Web Developer, specializing in JavaScript | GDSC Web Lead | Postman Student Expert",
+  "avatar": "https://github.com/grand-rick001.png",
```
Comment (sidharthviz) [inline data/grand-rick001.json:5]: Hi @grand-rick001  No need to add avatar, You can remove it

**Review by sidharthviz — CHANGES_REQUESTED** [review]

File: data/grand-rick001.json (hunk 1 above)
Comment (grand-rick001) [reply data/grand-rick001.json:5]: Yeah, just wanted to add a custom one to my LinkFree profile

Comment (mahfuztbt) [reply data/grand-rick001.json:5]: > 

looks like you add your github avatar, so in recent update your github profile pic will be add automatic, so you don't need `"avatar": "https://github.com/grand-rick001.png",`. you can remove it

Comment (grand-rick001) [reply data/grand-rick001.json:5]: Okay, on it

**Review by SurajPratap10 — APPROVED** [review]

**Comment by SaraJaoude** [comment]
There seem to be no changes in this Pull Request so I will be closing it.  Please raise a new one with your desired changes - you may wish to have a look at our Editing Guide in Docs: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request closed without merging**